This is an upstream patch for Apache Issue 49902, Fedora BZ#783742

Fix regression in 2.3.6 which broke process controls when using vhost-
specific configuration.

--- httpd/mod_fcgid/trunk/modules/fcgid/fcgid_spawn_ctl.c	2010/11/22 14:08:25	1037726
+++ httpd/mod_fcgid/trunk/modules/fcgid/fcgid_spawn_ctl.c	2010/11/22 14:08:29	1037727
@@ -178,7 +178,7 @@
         if (current_node->inode == command->inode
             && current_node->deviceid == command->deviceid
             && !strcmp(current_node->cmdline, command->cmdline)
-            && current_node->vhost_id == sconf->vhost_id
+            && current_node->vhost_id == command->vhost_id
             && current_node->uid == command->uid
             && current_node->gid == command->gid)
             break;
